src/main/java/com/feelmycode/parabole/service/CouponService.java

```diff
@@ -94,34 +86,35 @@ public Page<CouponUserResponseDto> getUserCouponList(Long userId) {
         List<CouponUserResponseDto> dtos = new ArrayList<>();
 
         for (int i = 0; i < couponList.size(); i++) {
-            UserCoupon uc = couponList.get(i);
-            Long cidOfUc = uc.getCoupon().getId();
-            Coupon c = couponRepository.findById(cidOfUc)
+            UserCoupon userCoupon = couponList.get(i);
+            Long cidOfUc = userCoupon.getCoupon().getId();
+            Coupon coupon = couponRepository.findById(cidOfUc)
                 .orElseThrow(() -> new ParaboleException(HttpStatus.BAD_REQUEST,
                     "쿠폰Id로 쿠폰을 검색한 내용이 존재하지 않습니다."));
 
-            dtos.add(new CouponUserResponseDto(c, uc, c.getSellerName()));
+            Seller seller = sellerRepository.findById(coupon.getSeller().getId()).orElseThrow();
+            dtos.add(new CouponUserResponseDto(coupon, userCoupon, seller.getUser().getName()));
         }
         return new PageImpl<>(dtos);
     }
 
     @Transactional(readOnly = true)
     public CouponAvailianceResponseDto getCouponInfo(String couponSNo) {
 
-        UserCoupon uc = userCouponRepository.findBySerialNoLike(couponSNo);
-        if (uc == null) {
+        UserCoupon userCoupon = userCouponRepository.findBySerialNo(couponSNo);
+        if (userCoupon == null) {
             throw new ParaboleException(HttpStatus.BAD_REQUEST,
                 "쿠폰 일련번호로 사용자 쿠폰을 검색한 내용이 존재하지 않습니다.");
         }
-        Coupon c = uc.getCoupon();
+        Coupon c = userCoupon.getCoupon();
 
         String type = null;
         Object ret = null;
 
-        if(c.getType() == CouponType.RATE){
+        if(c.getType() == 1){
```

Comment:
1로 비교 하는게 아니라 EnumType을 사용할거면 
```suggestion
        if(CouponType.isRATE(c)){
        if(CouponType.checkType(c))
```

Reply:
반영완입니다

Reply:
완료했엉요